packages/contracts-watr/contracts/TrueCurrency.sol

```diff
@@ -126,8 +130,9 @@ abstract contract TrueCurrency is BurnableTokenWithBounds {
         require(!isBlacklisted[recipient], "TrueCurrency: recipient is blacklisted");
 
         if (isRedemptionAddress(recipient)) {
-            super._transfer(sender, recipient, amount.sub(amount.mod(CENT)));
-            _burn(recipient, amount.sub(amount.mod(CENT)));
+            uint256 _amount = amount.sub(amount.mod(CENT));
```

Comment:
Good change, though we don't really need SafeMath here as it it always the case that for `uint256` that `amount >= amount mod CENT `. Not sure if we also want to avoid changes like this 🤔